css/style.css

```diff
@@ -121,6 +121,136 @@ ul {
   cursor: pointer;
 }
 
+.modal {
```

Comment:
- [OPTIONAL] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  kindly make this requested changes: 

- It would be better if the background is not scrollable when the popup page is open. you can use the `overflow:hidden;` CSS property to avoid this. 😄 
- Let's improve the  `See live and See source` buttons with their icons to macth the figma design 🎨 🖌️ 

Your Implementation | Figma Design
:---:|:---:
![Screenshot from 2023-05-17 23-59-13](https://github.com/ClaudiaRojasSoto/portfolio_1/assets/79058364/ab33a309-8da1-4a20-b61e-8110764ae118) | ![Screenshot from 2022-10-12 05-00-43](https://user-images.githubusercontent.com/79058364/195239977-37fb4625-af99-43e0-8509-0edd240c0d50.png)


- [OPTIONAL]  Please do the same for the desktop size 👨🏻‍🎨 🎨 


Your Implementation | Figma Design
:---:|:---:
![image](https://github.com/ClaudiaRojasSoto/portfolio_1/assets/79058364/5e12c71d-d8c0-490e-ba18-3840a0374f4e) | ![Screenshot from 2022-10-12 05-13-25](https://user-images.githubusercontent.com/79058364/195241407-f696f0f5-78d1-4853-b3ea-baeebab49b3d.png)